xbmc/utils/StringUtils.cpp

```diff
@@ -354,6 +355,18 @@ wchar_t toupperUnicode(const wchar_t& c)
   return c;
 }
 
+template <typename Type = char> struct toCapitalize
+{
+   toCapitalize() : bFirst(true) {}
+
+   Type operator()(Type c)
+   {
+      return std::isspace(c) ? (bFirst = true, c) : bFirst ? (bFirst = false, std::toupper(c)) : c;
```

Comment:
this doesn't compile, std::isspace and std::toupper takes two parameters, you probably meant ::isspace and ::toupper to access the c version of them